integ-test/src/test/resources/text_keyword_index.json

```diff
@@ -0,0 +1,16 @@
+{"index": {}}
+{"typeKeyword": "key00", "typeText": "text00", "typeKeywordFieldData": "keyFD00", "typeTextFieldData":  "textFD00", "int0": 0}
```

Comment:
none of these documents have text fields larger than 256, so we aren't testing the ignore_above sub-field. Can we do that?  Can we create a field with ignore_above of around 10, and add data (with spaces) larger than 10 characters?

Reply:
2a8fbacd599f31607935ba849ec4cf89527d9fb5 Added and it passes.